### ISSUE TITLE
Adding an easily accessible widget to set the current profile

### ISSUE THREAD

**Issue by wondie** [gltn/stdm#182]
This is based on what @pgathogo suggested to adding an easily accessible widget to set the current profile. 

I feel the best place for this widget is in the status bar in place of the current STDM Profile status message. So that message will be a combobox as shown in the screen shot below. 

![image](https://cloud.githubusercontent.com/assets/9709624/18882479/fce67cdc-84e7-11e6-8865-9c2c1a0a9f38.png)

A button that acts as apply will be added to change the current profile. This prevents accidental switch.

**Comment by gkahiu** [comment]
Good idea but why not place the combo box in the STDM toolbar?

**Comment by wondie** [comment]
That would have been good. However, that will make the toolbar to be very long. As a result, lead to the hiding of more buttons in small displays and in an over-crowded toolbar area.

**Comment by gkahiu** [comment]
The function of the status bar is just for displaying the 'status' and not
for changing options or system settings. Regarding the issue of being long,
depending on the placement QGIS will adjust it accordingly and insert
additional arrows at the end to indicate that there are more tools. Users
can also change the placement manually.

**Comment by wondie** [comment]
I agree with using the status bar for status only even though QGIS is using
it for more than a status.

However, making the menu will hide very frequently used buttons such as
logout and help. Hidden menus are not recommended by UI experts for
frequently used items.
https://msdn.microsoft.com/en-us/library/windows/desktop/dn742479(v=vs.85).aspx#controls

According to research, they are more difficult to find for the user and
less frequently used.

What about adding a small dock widget below the spatial unit manager?
